app/common/config.py

```diff
@@ -22,6 +22,8 @@
 MQ_URL: str = getenv("MQ_URL", "amqp://guest:guest@localhost/")
 MQ_POCHTA_QUEUE: str = getenv("MQ_POCHTA_QUEUE", "pochta.send")
 
+DEMO_DISWEBHOOK_URL: str = getenv("DEMO_WEBHOOK_URL", "https://")
```

Comment:
question: Why does the env-variable name start with `DEMO`? I think it can be confused with `default`-variable